pinot-spi/src/main/java/org/apache/pinot/spi/stream/PartitionLagInfo.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.spi.stream;
+
+public class PartitionLagInfo {
+  private String _offsetLag = "UNKNOWN";
+
+  public static PartitionLagInfo empty() {
+    return new PartitionLagInfo();
+  }
+
+  public String getOffsetLag() {
+    return _offsetLag;
```

Comment:
There should be an element of time in this. Not all offsets are numerical, so the underlying plugin can compute the time difference between the offsets presented.

In Kafka the offset difference neither equates to time nor number of messages